plugins/github-pull-requests-board/src/hooks/usePullRequestsByTeam.tsx

```diff
@@ -43,18 +47,52 @@ export function usePullRequestsByTeam(
 
       return pullRequestsWithDetails;
     },
-    [getPullRequests, getPullRequestDetails, pullRequestLimit],
+    [getPullRequestsFromRepository, getPullRequestDetails, pullRequestLimit],
+  );
+
+  const getPRsPerTeamMember = useCallback(
+    async (
+      teamMember: string,
+      teamOrganization?: string,
+    ): Promise<PullRequests> => {
+      const pullRequestsNumbers = await getPullRequestsFromUser(
+        teamMember,
+        teamOrganization,
+        pullRequestLimit,
+      );
+
+      const pullRequestsWithDetails = await Promise.all(
+        pullRequestsNumbers.map(({ node }) =>
```

Comment:
does this explode / run into rate limits if the number of PRs is large? would a p-limit limiter on this make sense?